### PR TITLE
Configure Sejong testnet

### PR DESCRIPTION
**Pull request by rise-techiast**
- [ ] Support IUSDC

### CODE REVIEW

**Review by BennyOptions — APPROVED** [review]
i tested all primary features and it works

File: src/packages/BalancedJs/addresses.ts [hunk 1]
```diff
@@ -51,28 +51,28 @@ const YEOUIDO_ADDRESSES = {
...
-  iusdc: 'cxb2c075e9130440dd64e6bfd3fe09e5c629f6e183',
-  iusdt: '',
-  usds: 'cx2e008873de78f5bf82008a46b529d94924d9b6ae',
-  cft: '',
-  metx: '',
-  bwt: 'cxa2afb37647ae91ca6fbf35141b2ef2ac7105720a',
+  sicx: 'cx70806fdfa274fe12ab61f1f98c5a7a1409a0c108',
+  bnusd: 'cx5838cb516d6156a060f90e9a3de92381331ff024',
+  baln: 'cx303470dbc10e5b4ab8831a61dbe00f75db10c38b',
+  // omm: '',
```
Comment (octocat512) [inline src/packages/BalancedJs/addresses.ts:69]: please remove the unused addresses.

**Review by octocat512 — CHANGES_REQUESTED** [review]

File: src/packages/BalancedJs/addresses.ts (hunk 1 above)
Comment (rise-techiast) [reply src/packages/BalancedJs/addresses.ts:69]: I'd prefer to leave it there for future updates, for example we're waiting to get OMM's Sejong contract

Comment (octocat512) [reply src/packages/BalancedJs/addresses.ts:69]: For the OMM token address, you have to change the token address in the constant tokens variable in the future.
There will be only baln-related tokens in the addresses variable.

Comment (octocat512) [reply src/packages/BalancedJs/addresses.ts:69]: image
In the addresses variable, the frontend only will save the addresses.

Comment (rise-techiast) [reply src/packages/BalancedJs/addresses.ts:69]: Ah I see, gotcha I'll update now

Comment (octocat512) [reply src/packages/BalancedJs/addresses.ts]: To add the new token address(for example, GBET), we had to add the token addresses in the addresses constant variable and the tokens constant variable. It was annoying for us. So we refactored.

**Comment by octocat512** [comment]
Please resolve the conflicts. after that, I will merge into master.

**Comment by rise-techiast** [comment]
> Please resolve the conflicts. after that, I will merge into master.

@octocat512 I've resolved the conflicts, let's merge now

**Pull request merged**